### ISSUE TITLE
Clearly make confirmation UI changes optional

### ISSUE THREAD

**Issue by simon-friedberger** [WICG/PEPC#21]
https://github.com/WICG/PEPC/blob/main/explainer.md#separate-this-into-two-proposals-1-improved-user-intent-signal-and-2-modal-permission-prompts notes
> We believe these aspects of the proposal offer the most user utility when bundled. If we only improve the user intent signal with a permission element, we fail to solve for change blindness and accessibility problems for magnification users. If we only introduce modal permission prompts without improving our confidence in user intent and context we increase the level of interruption and disruption in user journeys with blocking modals about which the user may have little or no context for decision making.

It is true that allowing websites to ask for permissions repeatedly requires a stronger signal of user intent.

On the other _If we only improve the user intent signal with a permission element, we fail to solve for change blindness and accessibility problems for magnification users._ doesn't mean that this needs a PEPC. This is an orthogonal problem.

PEPC should not prescribe a UI for asking users for permission.

**Comment by andypaicu** [comment]
I don't think PEPC prescribes a UI, the screenshots present an example of how the UI might look like, to help people visualize the proposal and its advantages better. However, knowing where users are currently focused allows users agents to provide better accessible UI. The paragraph is to present PEPC as a possible solution to specifically users that use magnification tools, I don't think it's presenting it as the only solution, or the only use case. Solving orthogonal problems doesn't seem like a bad thing to me.

**Comment by simon-friedberger** [comment]
So, it's only prescribing modal prompts or nothing?

If you don't think it prescribes a UI just clearly state that and I'm happy!

**Comment by andypaicu** [comment]
It doesn't prescribe any prompt UI at all. I believe all the UI screenshots given are preceded by text like "An example of how this could look:" or similar.

**Comment by simon-friedberger** [comment]
Sorry but I really don't understand. The section I linked to specifically addresses a previous suggestion "Separate this into two proposals, (1) improved user intent signal and (2) modal permission prompts". Are you saying (2) is no longer part of the proposal?

**Comment by andypaicu** [comment]
Honestly I think the title section is just misworded. Chrome does plan to use this to make PEPC-triggered prompts modal, but that's not relevant to the explainer and up to each user agent's implementation. Let me reword it.
